activitypub/inbox/delete.go

```diff
@@ -0,0 +1,25 @@
+package inbox
+
+import (
+	"context"
+
+	"github.com/go-fed/activity/streams/vocab"
+	"github.com/owncast/owncast/activitypub/persistence"
+	"github.com/owncast/owncast/activitypub/resolvers"
+	log "github.com/sirupsen/logrus"
+)
+
+func handleDeleteRequest(c context.Context, activity vocab.ActivityStreamsDelete) error {
+	// Only need to handle deletes for followers
+	if !activity.GetActivityStreamsObject().At(0).IsActivityStreamsPerson() {
+		return nil
+	}
+
+	actor, err := resolvers.GetResolvedActorFromActorProperty(activity.GetActivityStreamsActor())
```

Comment:
Since all we want to do is remove the follow it seems like doing an additional resolution of the actor's details isn't needed. Maybe we could add a `RemoveFollowIRI()` and just pass in the IRI instead of an entire actor?

Reply:
Yeah makes sense, I'll do that. Thanks!